esphome/components/modbus_controller/modbus_controller.h

```diff
@@ -281,8 +280,12 @@ class SensorItemsComparator {
       return lhs->start_address < rhs->start_address;
     }
 
-    // The pointer to the sensor are added at last to ensure that
-    // multiple sensors with the save values can be added with a stable sort order.
+    // sort by bitmask address
```

Comment:
Can you explain, why sorting by bitmask is important? For my understanding it does not play a role. Please note the last comparison on the return statement ensures that two sensors with the same address, function etc. can be added to the set.

Reply:
I had scenario where it did matter but to be honest can